Chambers_Jess/Assignments/Multiples_Part_2.py

```diff
@@ -0,0 +1,3 @@
+for x in range(5,1000000):
```

Comment:
good.

check the range() documentation, there may be an even _simpler_ way to do this!